drivers/serial/uart_stm32.c

```diff
@@ -665,6 +671,13 @@ static int uart_stm32_init(struct device *dev)
 
 	LL_USART_Disable(UartInstance);
 
+#if defined(CONFIG_LPUART_1) \
+	&& defined(CONFIG_SOC_SERIES_STM32G4X) \
+	&& defined(CONFIG_UART_CONSOLE)
+	/* the clock source for LPUART console is set to HSI */
+	LL_APB1_GRP2_EnableClock(LL_APB1_GRP2_PERIPH_LPUART1);
```

Comment:
Should be done by stm32_clock_control_on.

Reply:
already done in stm32_clock_control_on, so removed there